crates/frontend/src/components/dropdown.rs

```diff
@@ -66,7 +68,16 @@ where
             class=("dropdown-top", dropdown_direction == DropdownDirection::Top)
             class=("dropdown-down", dropdown_direction == DropdownDirection::Down)
         >
-            <label tabindex="0" class=format!("{} {}", class, btn_class)>
+            <label
+                tabindex="0"
+                class=format!("{} {}", class, btn_class)
+                on:click:undelegated=move |_| {
```

Comment:
also, how was this supposed to work ?

Reply:
Didn't get your question

Reply:
i was asking how was this change helping in solving the autofocus issue, couldn't understand the logic by looking at the code

Reply:
To further add on this, the component library we are using keeps the dropdown mounted on the DOM, so autofocus do focus on the input box but since we make other interactions that focus is lost.

Reply:
Other way of doing this can be taking over the logic to display dropdown using signals, but I don't want to manually manage close for any click outside of dropdown.

Reply:
> keeps the dropdown mounted on the DOM

why is it always mounted ?
shouldn't we just unmount it when not in use
anyways, the behaviour which we simply wanted is the moment it becomes visible (gets mounted in ideal case), the input component should get focused

Reply:
> shouldn't we just unmount it when not in use

Ideally yes.

> why is it always mounted ?

That's how daisy UI does it.
It just changes the visibility to hidden and opacity to 0.